src/tiled-map-resource.ts

```diff
@@ -421,28 +423,30 @@ export class TiledMapResource implements Loadable<TiledMap> {
       }
 
       // Create Excalibur sprites for each cell
-      for (var layer of this.data.rawMap.layers) {
-         if (layer.type === "tilelayer") {
-            for (var i = 0; i < layer.data.length; i++) {
-               let gid = <number>layer.data[i];
-
+      for (var rawLayer of this.data.rawMap.layers) {
+         if (rawLayer.type === "tilelayer") {
+            const layer = new TileMap(0, 0, this.data.rawMap.tilewidth, this.data.rawMap.tileheight, this.data.height, this.data.width);
+            const zindex = getProperty<number>(rawLayer.properties, 'zindex')?.value || -1;
```

Comment:
Now that we have separate layers a good default might be to assign a z in layer order, and use the custom 'zindex' property as an override
```suggestion
            const zindex = getProperty<number>(rawLayer.properties, 'zindex')?.value || layerZIndexBase++;
```

Reply:
@eonarheim Where do we want to start with the zindex?
Until now, the default value was -1. If we start at -1, then some layers are by default above the hero if no z-index is set.